usr/share/doc/zabbix-check-sshkey/zabbix_check_sshkey_template.xml

```diff
@@ -1,78 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why do we delete them definitively? Shouldn't we include a recent example?

Reply:
> You can review it, but don't merge it yet. I have to create and add a new template before.

Yes, I want to add a new one, that's why I tagged it as WIP and wrote this comment yesterday ;)